.github/workflows/deploy-storybook.yaml

```
@@ -0,0 +1,55 @@
+name: Deploy Storybook
+on:
+  pull_request:
+    types:
+      - closed
+    paths:
+      - '**.stories.tsx'
+      - '**.stories.ts'
+      - '**.mdx'
+
+jobs:
+  deploy-storybook:
+    if: github.event.pull_request.merged == true && github.event.pull_request.base.ref == 'main'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '20'
+
+      - name: install pnpm
+        run: npm i -g pnpm
+
+      - name: Cache node modules
+        id: cache-node
+        uses: actions/cache@v3
+        with:
+          path: |
+            **/node_modules
+          key: ${{ runner.os }}-node-${{ hashFiles('**/pnpm-lock.yaml') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
```

요부분은 추후에 이렇게 수정하면 더 간단해 질 것 같아요! 이렇게 [복합 액션](https://github.com/Central-MakeUs/PillME-Web/blob/main/.github/actions/pnpm-setup-node/action.yml)으로 만들어도 좋구여

그러고보니까 저희 nvmrc를 안 만들었네요. 노드 버전 고정시키는게 나중에 락파일에 영향을 안줄 것 같아서 추후 다른 PR에서 추가하면 좋을 것 같습니다.

```suggestion
      - name: pnpm setup
         uses: pnpm/action-setup@v4
         
       - name: node setup  
          uses: actions/setup-node@v4
          with:
            cache: 'pnpm'
            cache-dependency-path: 'pnpm-lock.yaml'
            node-version-file: '.nvmrc'
```